datagenerator/src/main/java/com/linkedin/restli/datagenerator/csharp/CSharpComplexType.java

```diff
@@ -0,0 +1,53 @@
+package com.linkedin.restli.datagenerator.csharp;
```

Comment:
For non-generated files, please add the following copyright boilerplate:

/*
   Copyright (c) 2017 LinkedIn Corp.

   Licensed under the Apache License, Version 2.0 (the "License");
   you may not use this file except in compliance with the License.
   You may obtain a copy of the License at

       http://www.apache.org/licenses/LICENSE-2.0

   Unless required by applicable law or agreed to in writing, software
   distributed under the License is distributed on an "AS IS" BASIS,
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
 */